src/network/src/esp/include/TCPServer.h

```diff
@@ -0,0 +1,36 @@
+#ifndef HIVE_CONNECT_TCPSERVER_H
+#define HIVE_CONNECT_TCPSERVER_H
+
+#include "BaseTask.h"
+#include "INetworkDeserializer.h"
+#include "Mutex.h"
+#include "c-common/circular_buff.h"
+#include "logger/Logger.h"
+#include <array>
+
+constexpr uint16_t g_MAX_BUFFER_SIZE = 1024;
```

Comment:
Is the g_ really needed? If we are in all caps, we know this is a constant

Reply:
Required by clang